docs(faq, glossary): minor fixes (#2)
joinmarket-webui/jamdocs: docs/glossary.md

### Collaborative Transaction

A collaborative transaction is a bitcoin transaction that --- you guessed it ---

> is done collaboratively by multiple participants. Specifically a coinjoin is a 

Reviewer: ```suggestion
is done collaboratively by multiple participants. Specifically, a coinjoin is a 
```

[orderbook]: market/orderbook.md

### Timechain
A timestamped and linked list of blocks. Anyone can create a new block, but each block must have 

> sufficient proof of work, making it difficult to create new blocks.

Reviewer: ```suggestion
sufficient proof of work, making it costly to create new blocks.
```

### Timelock

> A utxo can be locked up by a script which defines that this coin can only be spent in a block that 

Reviewer: ```suggestion
A UTXO can be locked up by a script which defines that said UTXO can only be spent in a block that 
```

> is higher than a certain value. This makes coins unspendable before a specific time.

Reviewer: ```suggestion
is higher than a certain value. This makes UTXOs unspendable before a specific time.
```

## U
### Unspent Transaction Output

> The tip of the chain of signatures which originates in a coinbase output. It's a coin that can 

Reviewer: ```suggestion
The tip of the chain of signatures which originates in a coinbase output. It's a "coin" that
```
